chef/cookbooks/ceilometer/recipes/server_ha.rb

```diff
@@ -60,9 +60,17 @@
 if node[:ceilometer][:use_mongodb]
   pacemaker_order "o-ceilometer-mongo" do
     score "Mandatory"
```

Comment:
I'm unsure if we want to keep it Mandatory. That sounds wrong given the fact that there are many daemons in the group that might still just work fine if mongodb goes down.